mne/viz/tests/test_epochs.py

```diff
@@ -100,6 +100,15 @@ def test_plot_epochs():
     plt.close('all')
 
 
+def test_plot_concat():
+    """ Test concatenated epoch plotting
+    """
+    import matplotlib.pyplot as plt
```

Comment:
I think we do for `plt`, actually. Not for `matploltib` since we need to set the backend. I can't remember why we made the distinction offhand.